Lib/inspect.py

```diff
@@ -49,7 +49,7 @@
 import functools
 import builtins
 from operator import attrgetter
-from collections import namedtuple, OrderedDict
+from collections import namedtuple
```

Comment:
If inspect needs to import collections anyway, I don’t think there is much value in removing OrderedDict.
Also replacing `params.move_to_end(p)` by `params[p] = params.pop(p)` without even a comment makes the code a bit worse to read.

I would be -1 on this PR.

(I thought I already left this comment?  Are there duplicate PRs?)